priv/template/lib/app_name/application.ex.eex

```diff
@@ -15,6 +15,9 @@ defmodule <%= @module %>.Application do
     ]
 
     children = [
+      <%= if @ecto do %>
```

Comment:
Can this all go on one line?

otherwise i think you will get a random newline in the case when ecto flag is false

Reply:
Can do, I'm still not too proficient in thinking with templates ;)

Reply:
I was wondering if I could run the formatter on the new project. then these issues would go away. separate issue though

Reply:
I like this idea. Let's have it as a separate issue and see if it's simple to do.